src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -15,7 +15,11 @@ spec:
     metadata:
       annotations:
         checksum/config-jvm: {{ include (print $.Template.BasePath "/synchrony-start-script.yaml") . | sha256sum }}
+        {{- if .Values.synchrony.podAnnotations }}
```

Comment:
When this is evaluated not true - is this false for `.Values.synchrony.podAnnotations == {}`?

Reply:
yes, undefined podAnnotations will make `{{- if .Values.synchrony.podAnnotations }}` appear as false